Xamarin.Forms.ControlGallery.Uno.Wasm/Xamarin.Forms.ControlGallery.Uno.Wasm.csproj

```diff
@@ -32,10 +32,10 @@
 	</ItemGroup>
 
 	<ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.1.0.709244" ExcludeAssets="build" />
+    <PackageReference Include="Xamarin.Forms" Version="4.2.0.709249" ExcludeAssets="build" />
```

Comment:
Should not have been updated

Reply:
Hoo okay it's was from @gHuntley i understood that i should update Xamarin.Forms too.
Should I Revert back?